docs(async-csv): Add instructions for checking SHA1 checksum on CSV (#1747)
getsentry/sentry-docs: src/collections/_documentation/performance/discover/query-builder.md

### Download CSV

> If you want to take the data elsewhere, click on the "Export" button to start the export. The results are going to take a while. You'll get an email with the download link once they're ready.

Reviewer: Can we say "depending on the file size, the results can take between one second to one hour"

Author: Would that be misleading if the results can take well over an hour? I've seen it take like 90 minutes, and there isn't really a strict upper bound as how long it'll take so it can potentially go above that.

Reviewer: How about:

Depending on the file size, the result times will vary.

Reviewer: Is that statement accurate? I'm assuming the results are dependent on the file size.

Author: Can we can say

Depending on the amount of data, the wait times will vary.

Reviewer: 👍 That's great! Thanks, Tony

Author: ```suggestion
If you want to take the data elsewhere, click on the "Export" button to start the export. Depending on the amount of data, the wait times will vary. You'll get an email with the download link once they're ready.
```

> If you want to take the data elsewhere, click on the "Export" button to start the export. The results are going to take a while, so sit back and relax. You'll get an email with the download link once they're ready.

Reviewer: :) Let's be more specific about how long this will take. Do we have an estimation of time?

Reviewer: Also, our docs have a more formal tone. Let's remove the informal voice here.
```suggestion
If you want to take the data elsewhere, click on the "Export" button to start the export. The results are going to take a while. You'll get an email with the download link once they're ready.
```

Author: Unfortunately this can be anywhere from 1 second to over an hour, so I don't have a good estimate to put here.

{% capture __alert_content -%}

> The results are limited to 10 million rows or 2GB whichever comes first.

Reviewer: ```suggestion
The results are limited to 10 million rows or 2GB, whichever comes first.
```

{%- endcapture -%}
{%- include components/alert.html
    title="Note"
    content=__alert_content
    level="info"
%}

#### Verifying the Download

> The download page contains a `SHA1` checksum of the file which you can use to verify its integrity with the following command.

Reviewer: ```suggestion
The download page contains a `SHA1` checksum of the file, which you can use to verify its integrity with the following command:
```
